sdk/core/azure-core/azure/core/pipeline/__init__.py

```diff
@@ -101,7 +98,7 @@ def pop(self, *args):
         return super(PipelineContext, self).pop(*args)
 
 
-class PipelineRequest(Generic[HTTPRequestType]):
+class PipelineRequest():
```

Comment:
No, a PipelineRequest is a generic object, that can contains any types, we didn't force the types to HttpRequest on purpose